### PR TITLE
Adds NCR rep

### PR DESCRIPTION
**Pull request by printer16**
## About The Pull Request
As requested, added the NCR rep and their loadouts. If there's any suggestions or concerns regarding their load out let me know.

## Why It's Good For The Game

As requested, added the NCR rep and their loadouts. If there's any suggestions or concerns regarding their load out let me know. They are not mapped in yet, that's in a future PR.

## Pre-Merge Checklist
- [X] Your Pull Request contains no breaking changes
- [X] You tested your changes locally, and they work.
- [X] There are no new Runtimes that appear.
- [X] You documented all of your changes.

## Changelog
:cl:
add: Adds the NCR rep.

/:cl:

### CODE REVIEW

**Comment by printer16** [comment]
Don't merge just yet

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -347,6 +347,73 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
+		/datum/matchmaking_pref/rival = list(
+			/datum/job/ncr
+			)
+		)
+
+/datum/outfit/job/ncr/f13representative
+	name = "NCR Representative"
+	uniform = /obj/item/clothing/under/f13/ncr/ncr_dress
+	jobtype	= /datum/job/ncr/f13representative
+	id = /obj/item/card/id/dogtag/ncrtrooper
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:379]: I suggest replacing the NCR dogtag with an Ambassador ID, to show a little difference

**Comment by BadAtThisGame302** [comment]
I suggest replacing the NCR dogtag with an Ambassador ID, to show a little difference

**Comment by printer16** [comment]
Should be good.

**Comment by hecksadecimal** [comment]
This will be reverted, and then re-added when the administrative and lore teams deem it ready for that. 
You won't have to re-PR this, I can just revert the revert.

**Pull request merged**